### DISCUSSION TITLE
Referendum Media Tsunami

### DISCUSSION THREAD

**Discussion by lvlast3l2** [WeAreMahsaAmini/Core#189] [Ideas]
یه پلتفورمی ایجاد بشه، افراد هر کسی به صورت یک نفر یک نفر، مقابل دوربین قرار بگیره و یه جمله مشخص و واحدی رو تکرار کنه. مثلا، زن، زندگی، آزادی

هر کسی موافق آزادی هست، ویدیو هاشونو ارسال میکنه. 

نکته مهم

این ویدیو ها، تا قبل از 20میلیون شدن ، **منتشر نخواهند شد.** 

قسمت چشم های افراد پیش از انتشار، یه هایلایتی قرار بگیره که قابل شناسایی نباشن پیش از انتشار ویدیو ها.

 بعد از اینکه ویدیو ها منتشر شد،  دیگه حکومت با 20 میلیون نفر یکجا نمیتونه برخورد کنه. موقعی که بازخورد از حد بگذره، تبدیل یه حرکت نمادین و اجتماعی بزرگ میشه. اصطلاح بهترش، سونامی اجتماعی. دیگه موج کوچیکی نیست که بشه مقابلش با سفسفته و دروغ و وعده وعید ایستاد. 

20میلیون رای میشه.. 
12 فروردین 1357 با 98% رای مثبت مردم توی رفراندوم جمهوری اسلامی، با 20.309.595  رای مثبت انتخاب شد

بماند که من نظر شخصیم 40میلیون نفر هست. چون ایرانی های خارج از ایران هم میتونن شرکت کنند. غیر ایرانی ها هم میتونن شرکت کنند. نهایتا افراد غیر ایرانی ، جداگانه ویدیوشون ذخیره بشه. فقط اگر بتونید امنیت رو تامین کنید، مطمئنم خیلی از افراد ، **چه موافق  و چه مخالف**، بعد از این بی فکری حکومت، صداشون در اومده و فقط نگران امنیت هستن.

پس از رسیدن به تعداد مشخص، در قالب یک مستند یا یک سرویس اشتراک ویدیو برای رای گیری منتشر بشه.

این سونامی، با هدف آزادی انتخاب مردم و شنیده شدن خواسته های مردم قابل مهار نخواهد بود و این کاریه که فقط شما میتونید عملیش کنید

**Comment by Atabak20** [comment]
Benazaram ideye kheyli khubie
